src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1514,6 +1514,18 @@ public override BoundNode VisitAssignmentOperator(BoundAssignmentOperator node)
             return null;
         }
 
+        public override BoundNode VisitDeconstructionAssignmentOperator(BoundDeconstructionAssignmentOperator node)
+        {
+            foreach (BoundExpression variable in node.LeftVariables)
+            {
+                VisitLvalue(variable);
```

Comment:
I assume properties could be used on the left side Perhaps the logic from VisitCompoundAssignmentOperator should be duplicated in some form here.